README.md

```diff
@@ -1,66 +1,128 @@
-# xCORE VocalFusion 4-Mic Kit for AVS and Amazon Alexa Voice Services SDK on a Raspberry Pi
+# xCORE VocalFusion Kit for Amazon AVS on a Raspberry Pi
 
-The XMOS **xCORE VocalFusion 4-Mic Kit for AVS** provides far-field voice capture using the XMOS XVF3000 voice processor.
+The XMOS **xCORE VocalFusion Kit for Amazon AVS** provides far-field voice capture using the XMOS XVF3100, XVF3500, and XVF3510 voice processors.
 
 Combined with a Raspberry Pi running the Amazon Alexa Voice Service (AVS) Software Development Kit (SDK), this kit allows you to quickly prototype and evaluate talking with Alexa.
 
 To find out more, visit: https://xmos.com/vocalfusion-avs  
 and: https://developer.amazon.com/alexa-voice-service
 
-This repository provides a simple-to-use automated script to install the Amazon AVS SDK on a Raspberry Pi and configure the Raspberry Pi to use the **xCORE VocalFusion 4-Mic Kit for AVS** for audio.
+This repository provides a simple-to-use automated script to install the Amazon AVS SDK on a Raspberry Pi and configure the Raspberry Pi to use the **xCORE VocalFusion Kit for Amazon AVS** for audio.
 
 ## Prerequisites
 You will need:
 
-- **xCORE VocalFusion 4-Mic Kit for Amazon AVS**: XK-VF3000-L33-AVS
-- Raspberry Pi 3
-- Micro-USB power supply (min. 2A)
+- Either
+
+   **xCORE VocalFusion XVF3510 Kit for Amazon AVS**: XK-VF3510-L71  
+   **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**: XK-VF3500-L33-AVS  
+   **xCORE VocalFusion 4-Mic Kit for Amazon AVS**: XK-VF3000-L33-AVS
+
+- Either
+
+   Raspberry Pi 3 with Micro-USB power supply (min. 2A) and HDMI cable
+   Raspberry Pi 4 with USB-C power supply (min. 3A) and Micro HDMI to Standard HDMI (A/M) Cable
+
 - MicroSD card (min. 16GB)
-- Powered mono speaker with audio 3.5mm analogue plug
+- Powered speakers with audio 3.5mm analogue plug
+
+   Stereo for the XVF3500 and XVF3500  
```

Comment:
The second XVF3500  should be XVF3510 I reckon

Reply:
Yup... Fixed.